Automattic/wp-calypso
# Phase Out WordPress Blue From the UI

This is a graceful step towards the bigger set of color changes coming to Calypso in order to bridge the visual gap between marketing pages and our product. The idea is to possibly minimize the color jumps.

### Before and After

<img width="1626" alt="Screen Shot" src="https://user-images.githubusercontent.com/2060580/80221947-5891bc00-8646-11ea-9287-1155f5e50937.png">

<img width="1626" alt="Screen Shot" src="https://user-images.githubusercontent.com/2060580/80221960-5b8cac80-8646-11ea-9ec7-4d9072b38141.png">

### Testing Instructions

* Pull the changes:
    * `git checkout update/blue`
    * `yarn`
* Run the SVG color audit script and make sure it doesn’t report any specific paths.
    * `node bin/audit-svg-colors.js`
* Preview the images listed by the command below (previewing locally should be enough) and make sure none of them throws any errors that would prevent them from displaying.
    * `git show HEAD~1 --name-only | grep 'svg$'`
* `yarn start`
* Open Calypso and make sure the master bar has a new color.
* Make sure the color scheme picker is using the updated thumbnails for Classic Blue and Classic Bright.

### Notes

* The illustrations will have to be handled in a separate PR.
